### ISSUE TITLE
Message configuration

### ISSUE THREAD

**Issue by simplylate** [arcadiadevs/playerservers-everything#13]
Not every message in PlayerServers is configurable which can lead to inconsistency with colour schemes.

**Comment by OpenSource03** [comment]
Yes, we're aware and we will put special attention into this in the future versions.
